index.js

```diff
@@ -13,6 +8,10 @@ function mewt (target) {
   }
 
   const override = prop => (...args) => {
+    const multiRet = 'push pop shift unshift'
```

Comment:
In this particular case there is no size difference with the minifier, so that is not in consideration.

The `multiRet` `mutArrMethods` and `nonMutArrMethods` are defined on the outer scope, but only used in the inner override scope. IMO, there is no need to do this as you can simply place the variables where they are needed and closer to where they are used. But, it's a style choice. I don't think this is necessarily a an example of a closure memory leak, but this style where you define variables outside of a function, only to use it inside can create a memory leak under certain conditions. I try to avoid it where possible.